services.yaml

```diff
@@ -2123,12 +2123,18 @@ paths:
                 items:
                   type: object
                   properties:
-                    type:
+                    name:
                       type: string
-                      example: "event"
-                    id:
+                      example: "Main.main"
```

Comment:
Are we sure this is not with the weird `Main#main#()V`?

Reply:
fixed